### PR TITLE
update scala client dependency

### PR DESCRIPTION
**Pull request by piuccio**
needed for nhs special report.

defaults do not apply anymore to stories that are both special features and comments

@OliverJAsh

### CODE REVIEW

**Comment by OliverJAsh** [comment]
Looks good to me but the build failed, and I don't have the context someone like @janua has :-)

**Comment by piuccio** [comment]
failed because of unresolved dependency gu:fapi-client_2.11:0.68 might take some time to be published

**Comment by janua** [comment]
Run it again soon!

File: app/util/ContentUpgrade.scala [hunk 1]
```diff
@@ -28,7 +31,7 @@ object ContentUpgrade {
   def upgradeResponse(json: JValue) = {
     json \ "response" match {
       case jsObject: JObject =>
-        JObject("response" -> (jsObject update JObject(ContentFields flatMap { field =>
+        JObject("response" -> (jsObject ~ JObject(ContentFields flatMap { field =>
```
Comment (janua) [inline app/util/ContentUpgrade.scala]: Why use `~` over `update`?

Comment (piuccio) [inline app/util/ContentUpgrade.scala]: becasue `~` is in the native DSL, `update` was created by Rob as an implicit and had this comment

```
/** I really thought this method would exist in Json4s but I couldn't find it */
```

Comment (janua) [inline app/util/ContentUpgrade.scala]: Thanks for JSON4S @robertberry

**Comment by janua** [comment]
Looks good! :+1:

**Comment by OliverJAsh** [comment]
Hey, do you think this will be released today? Just so I can let editorial
know.

On Mon, 18 Jan 2016, 15:34 Francis <notifications@github.com> wrote:

> Looks good! [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/facia-tool/pull/113#issuecomment-172562899>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Pull request merged**